cmd/helm/plugin_update.go

```diff
@@ -49,14 +55,22 @@ func newPluginUpdateCmd(out io.Writer) *cobra.Command {
 			return o.run(out)
 		},
 	}
+	cmd.Flags().StringVar(&o.version, "version", "", "specify a version constraint. If this is not specified, the latest version is installed")
```

Comment:
I'm a little confused by this flag. You have a flag and `version` added to the `pluginUpdateOptions` struct but you aren't using it. Should this be removed?

Reply:
I added this flag just because I wanted it to be shown under list of flags when performing  `helm plugin update -h`. Will remove it.